panedr/panedr.py

```diff
@@ -323,6 +329,20 @@ def add_blocks(self, final_number):
             self.nblock_alloc = final_number
 
 
+class GMX_Unpacker(xdrlib.Unpacker):
+    """xdrlib.Unpacker subclass that implements `unpack_real`
+
+    Decision on whether to return 32- or 64-bit reals is controlled by the
+    `gmx_double` attribute, set to ``False`` by default.
+    """
+    gmx_double = False
```

Comment:
I thought about that and in the end classattr won:

Pros:
- I don't need an `__init__` to set the default value of `gmx_double`. This is cleaner because `xdrlib.Unpacker` is old-style and calling its `__init__` from `GMX_Unpacker.__init__` has to be done without `super`. Not ideal.
- The code is shorter.

Cons:
- Semantically, I'm setting as a classattr the default value for something that's conceptually instance-bound. In practice it works as expected even if you open different precision files in a row because when I decide the precision I set the attr on the instance, clobbering the classattr.

A cool aspect is that, because precision is decided at each frame's reading time and changed on-the-fly, the code could now conceptually read mixed-precision files in which some frames are single and others double.